### DISCUSSION TITLE
pimctl not available

### DISCUSSION THREAD

**Discussion by kkatsaros-dcat** [troglobit/pimd#199] [Q&A]
I have downloaded and installed pimd on a Linux machice (Ubuntu 20.04) using the linux packages

`$ sudo apt install pimd`

It is successfully installed and it is running

```
$ pimd -v
pimd version 2.3.2
```

But when I try to use the pimctl tool to it can not find it. 
```
$ pimctl show neighbor
Command 'pimctl' not found,
```

Is there something I'm missing?

**Comment by troglobit** [comment]
This is the upstream pimd project, the documentation you see here pertains to that (yet to be released) new version, which is still not available pre-packaged in downstream distributions like Debian or Ubuntu.
